src/UI/Buyer/src/app/components/checkout/checkout/checkout.component.ts

```diff
@@ -260,9 +260,7 @@ export class OCMCheckout implements OnInit {
           this.order.ID,
           payment
         )
-        //  Must only patch order AFTER order has been submitted
-        //  to prevent order worksheet data from being cleared.
-        await this.patchSubmittedOrder(order, comment, payment)
+        await this.checkout.patch({Comments: comment}, order.ID)
```

Comment:
Can we add some comments either here or to the method to warn people not to try patching the order.xp because it resets order worksheet and to instead do it in the integration event?